From 76803c70cdf774d2de369714357ea6451b19fffb Mon Sep 17 00:00:00 2001
From: Johannes Berg <johannes.berg@intel.com>
Date: Thu, 30 Mar 2017 14:23:21 +0200
Subject: [PATCH 043/146] iwlwifi: mvm: link queue cmd docs to A000 command
 structs
Git-commit: e153c1e4c37fcffa26e901251767c1404db83934
Patch-mainline: v4.13-rc1
References: FATE#322675

Document which structures are used with the command for
the A000 hardware flavour.

Signed-off-by: Johannes Berg <johannes.berg@intel.com>
Signed-off-by: Luca Coelho <luciano.coelho@intel.com>
Signed-off-by: Oliver Neukum <oneukum@suse.com>
---
 drivers/net/wireless/intel/iwlwifi/mvm/fw-api.h | 4 +++-
 1 file changed, 3 insertions(+), 1 deletion(-)

diff --git a/drivers/net/wireless/intel/iwlwifi/mvm/fw-api.h b/drivers/net/wireless/intel/iwlwifi/mvm/fw-api.h
index 67b854c2a4d2..51afd99b38f5 100644
--- a/drivers/net/wireless/intel/iwlwifi/mvm/fw-api.h
+++ b/drivers/net/wireless/intel/iwlwifi/mvm/fw-api.h
@@ -232,7 +232,9 @@ enum iwl_legacy_cmds {
 
 	/* scheduler config */
 	/**
-	 * @SCD_QUEUE_CFG: &struct iwl_scd_txq_cfg_cmd
+	 * @SCD_QUEUE_CFG: &struct iwl_scd_txq_cfg_cmd for older hardware,
+	 *	&struct iwl_tx_queue_cfg_cmd with &struct iwl_tx_queue_cfg_rsp
+	 *	for newer (A000) hardware.
 	 */
 	SCD_QUEUE_CFG = 0x1d,
 
-- 
2.12.3

